client/editor/tool_tile.cpp

```diff
@@ -28,6 +31,8 @@ editor_tool_tile::editor_tool_tile(QWidget *parent)
   ui.setupUi(this);
 
   // Set default values and the font we want.
+  ui.value_continent->setText(_("N/A"));
```

Comment:
Side remark: not all instances of `N/A` translate the same, for instance in French I'd do:
* `tile: N/A` -> `tuile: aucune`
* `continent: N/A` -> `continent: aucun`

I translate to `aucun` ("none") because the French translation of N/A (`sans objet`) feels very formal and its abbreviation `s/o` is not very common.

Reply:
Maybe use "Not Set" instead?

Reply:
A dash would work. Maybe that's the solution for French :D

We may need a "click on a tile to view its properties" state for the whole widget?

Reply:
I could add a status label, good idea